pkg/kubelet/dockershim/helpers_unsupported.go

```diff
@@ -47,3 +47,11 @@ func (ds *dockerService) determinePodIPBySandboxID(uid string) string {
 	glog.Warningf("determinePodIPBySandboxID is unsupported in this build")
 	return ""
 }
+
+func getNetworkNamespace(c *dockertypes.ContainerJSON) (string, error) {
```

Comment:
This function is actually pretty Linux specific.  Might as well just return an error for _unsupported.

Reply:
This has been fixed in our latest commit